mem/mem_linux.go

```diff
@@ -296,15 +296,17 @@ func fillFromMeminfoWithContext(ctx context.Context) (*VirtualMemoryStat, *ExVir
 
 	ret.Cached += ret.Sreclaimable
 
-	if !memavail {
+	if memavail {
```

Comment:
I wonder if this should be opt-out/opt-in? Might be a breaking change for users, or if not breaking at least might be an unnoticed change that makes their stats look different all of a sudden.

The other stats are still available in `struct VirtualMemoryStat` so it's possible for consumers to redo the existing calculation on their own, but some manner of `ctx` option to revert to the old calculation might be a good idea?